src/main/java/org/eclipse/pathfinder/api/GraphTraversalService.java

```diff
@@ -23,36 +22,17 @@
 
     private static final long ONE_MIN_MS = 1000 * 60;
     private static final long ONE_DAY_MS = ONE_MIN_MS * 60 * 24;
-
-    @Inject private GraphDao dao;
     private final Random random = new Random();
+    @Inject private GraphDao dao;
 
     @GET
     @Path("/shortest-path")
     @Produces({"application/json", "application/xml; qs=.75"})
     public List<TransitPath> findShortestPath(
-            // TODO [Jakarta EE 8] Use not blank instead.
-            @NotNull(message = "Missing origin UN location code.")
-                    @Size(
-                            min = 5,
-                            max = 5,
-                            message = "Origin UN location code value must be five characters long.")
-                    @QueryParam("origin")
-                    String originUnLocode,
-            // TODO [Jakarta EE 8] Use not blank instead.
-            @NotNull(message = "Missing destination UN location code.")
-                    @Size(
-                            min = 5,
-                            max = 5,
-                            message =
-                                    "Destination UN location code value must be five characters long.")
-                    @QueryParam("destination")
-                    String destinationUnLocode,
-            // TODO [DDD] Apply regular expression validation.
-            @Size(min = 8, max = 8, message = "Deadline value must be eight characters long.")
-                    @QueryParam("deadline")
-                    String deadline) {
-        Date date = nextDate(new Date());
+            @NotNull @Size(min = 5, max = 5) @QueryParam("origin") String originUnLocode,
```

Comment:
This is a sync mismatch. Can you kindly restore the more updated code?

Reply:
restored.